Veritrans/Config.php

```diff
@@ -1,17 +1,47 @@
 <?php
-
+/**
+ * Veritrans Configuration
+ */
 class Veritrans_Config {
 
+  /**
+   * Your merchant's server key
+   * @static
+   */
   public static $serverKey;
+  /**
+   * Your merchant's client key
+   * @static
+   */
   public static $client_key;
+  /**
+   * This should be 2
```

Comment:
It's never used in code, but I think it suppose to change 'https://api.veritrans.co.id/v2' -> 'https://api.veritrans.co.id/v3'. I think better remove it